js/form.js

```diff
@@ -25,61 +33,22 @@
 
   const validateTitleInput = () => {
     const valueLength = adNode.title.value.length;
-    adNode.title = (valueLength < MIN_TITLE_SYMBOL_LENGTH) ? adNode.title.setCustomValidity(`Еще ${MIN_TITLE_SYMBOL_LENGTH - valueLength} символов!`) : adNode.title.setCustomValidity(``);
+    adNode.title = valueLength < MIN_TITLE_SYMBOL_LENGTH ? adNode.title.setCustomValidity(`Еще ${MIN_TITLE_SYMBOL_LENGTH - valueLength} символов!`) : adNode.title.setCustomValidity(``);
```

Comment:
Присваивание тут ничего не дает
```suggestion
    adNode.title.setCustomValidity( valueLength < MIN_TITLE_SYMBOL_LENGTH ? `Еще ${MIN_TITLE_SYMBOL_LENGTH - valueLength} символов!` : ``);
```